frontend/javascripts/oxalis/view/statusbar.tsx

```diff
@@ -211,7 +211,85 @@ function ShortcutsInfo() {
               Space
             </span>
           </span>{" "}
-          Trace forward
+          Trace forward +
```

Comment:
```suggestion
          Trace forward
```

I think, `forward` is enough to signal the direction.